bundles/binding/org.openhab.binding.vdr/README.md

```diff
@@ -6,12 +6,20 @@ If you have any suggestions or questions don't hesitate to contact me ([iwow](ht
 
 ## Binding Configuration
 
-This binding can be configured in the file `services/vdr.cfg`.
+This binding must be configured in the file `services/vdr.cfg` (OpenHAB 2) or `openhab.cfg` (OpenHAB 1).
```

Comment:
All the READMEs were revised a while back to use this same verbiage. It makes them consistent with each other, and since the audience still using OH1 is vanishingly small, it's not terribly useful to add text here for OH1 configuration.